sample-pkgr.yml

```diff
@@ -0,0 +1,47 @@
+# The version number of the config file, used to indicate the API for the config.
+# Do not change.
+Version: 1
+
+# Top-level packages. Include the packages you wish to have in your environment
+# Dependencies for these packages will be automatically determined and installed.
+Packages:
+  - pkg1
+  - pkg2
+  - pkg3
+
+# CRAN-Like Repositories from which packages will be downloaded.
+# The order that the repositories are listed in matters: Pkgr will look for
+# each package in the first repository listed. If a package is not found in the
+# first, Pkgr will check the second, then the third, etc.
+# first repository listed for each
+Repos:
+  - companyA_repo: "https://companyA.github.io/rpkgs"
+  - CRAN: "https://cran.rstudio.com"
+
+# Path the install packages to
+Library: "path/to/install/library"
+
+# Options for Logging
+# Without any options set, Pkgr will only log Info-level messages to the terminal.
```

Comment:
it will log info _and above_ level messages (warning/error etc) to standard output device  (generally terminal)